testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeDatabricksHttpHms.java

```diff
@@ -0,0 +1,116 @@
+
+    private final DockerFiles.ResourceProvider configDir;
+
+    @Inject
+    public EnvMultinodeDatabricksHttpHms(StandardMultinode standardMultinode, DockerFiles dockerFiles)
+    {
+        super(standardMultinode);
+        requireNonNull(dockerFiles, "dockerFiles is null");
+        configDir = dockerFiles.getDockerFilesHostDirectory("conf/environment/multinode-databricks-http-hms");
+    }
+
+    @Override
+    public void extendEnvironment(Environment.Builder builder)
+    {
+        String databricksTestJdbcUrl = requireNonNull(System.getenv("DATABRICKS_UNITY_JDBC_URL"), "Environment DATABRICKS_UNITY_JDBC_URL was not set");
+        String databricksTestLogin = requireNonNull(System.getenv("DATABRICKS_LOGIN"), "Environment DATABRICKS_LOGIN was not set");
+        String databricksTestToken = requireNonNull(System.getenv("DATABRICKS_TOKEN"), "Environment DATABRICKS_TOKEN was not set");
+        String awsRegion = requireNonNull(System.getenv("AWS_REGION"), "Environment AWS_REGION was not set");
+
+        builder.configureContainer(COORDINATOR, dockerContainer -> exportAWSCredentials(dockerContainer)
+                .withEnv("AWS_REGION", awsRegion)
+                .withEnv("DATABRICKS_TOKEN", databricksTestToken)
+                .withEnv("DATABRICKS_HOST", getEnvVariable("DATABRICKS_HOST"))
+                .withEnv("DATABRICKS_UNITY_CATALOG_NAME", getEnvVariable("DATABRICKS_UNITY_CATALOG_NAME")));
+
+        builder.configureContainer(WORKER, dockerContainer -> exportAWSCredentials(dockerContainer)
+                .withEnv("AWS_REGION", awsRegion)
+                .withEnv("DATABRICKS_TOKEN", databricksTestToken)
+                .withEnv("DATABRICKS_HOST", getEnvVariable("DATABRICKS_HOST"))
+                .withEnv("DATABRICKS_UNITY_CATALOG_NAME", getEnvVariable("DATABRICKS_UNITY_CATALOG_NAME")));
+
+        builder.configureContainer(TESTS, container -> exportAWSCredentials(container)
+                .withEnv("DATABRICKS_JDBC_URL", databricksTestJdbcUrl)
+                .withEnv("DATABRICKS_LOGIN", databricksTestLogin)
+                .withEnv("DATABRICKS_TOKEN", databricksTestToken)
+                .withEnv("DATABRICKS_UNITY_CATALOG_NAME", getEnvVariable("DATABRICKS_UNITY_CATALOG_NAME"))
+                .withEnv("DATABRICKS_UNITY_EXTERNAL_LOCATION", getEnvVariable("DATABRICKS_UNITY_EXTERNAL_LOCATION"))
+                .withCopyFileToContainer(
+                        forHostPath(DATABRICKS_JDBC_PROVIDER.getAbsolutePath()),
+                        "/docker/jdbc/databricks-jdbc.jar"));
+
+        builder.addConnector("hive", forHostPath(configDir.getPath("hive.properties")));
+        builder.addConnector(
+                "delta_lake",
+                forHostPath(configDir.getPath("delta.properties")),
+                CONTAINER_TRINO_ETC + "/catalog/delta.properties");
+        configureTempto(builder, configDir);
+    }
+
+    private DockerContainer exportAWSCredentials(DockerContainer container)
```

Comment:
nit: Rename AWS to Aws. Same for `exportAWSCredential`. Also, move this method to above `exportAWSCredential`. No need to have `getEnvVariable` method between these 2 methods.